README.md

```diff
@@ -1,9 +1,9 @@
 # Grails MongoDB Database Migration Plugin
-Latest Version (0.2.4)
+Latest Version (0.2.5)
```

Comment:
major release. Please update the version accordingly.

Reply:
Ok Changing it to 3.3.0 and before final review, let's discuss with @visheshd for versioning in all the plugins.